packages/postgres-database/scripts/erd/cli.py

```diff
@@ -0,0 +1,83 @@
+#
+# ERD (Entity Relationship Diagram) is used to visualize these relationships
+#
+# - Uses sqlalchemy_schemadisplay which is maintained by sqlalchemy
+# - DROPPED 'eralchemy' since it fails with latest version and is not maintained anymore
+#
+# SEE https://github.com/sqlalchemy/sqlalchemy/wiki/SchemaDisplay
+# SEE https://github.com/Alexis-benoist/eralchemy
+
+# pylint: disable=wildcard-import
+# pylint: disable=unused-wildcard-import
+
+
+import argparse
+import importlib
+import logging
+from pathlib import Path
+from typing import Any
+
+import simcore_postgres_database.models
+from simcore_postgres_database.models.base import metadata
+from sqlalchemy_schemadisplay import create_schema_graph
+
+models_folder = Path(simcore_postgres_database.models.__file__).parent
+# imports all models to fill "metadata"
+for p in models_folder.glob("*.py"):
+    if not p.name.startswith("__"):
+        importlib.import_module(
+            f"simcore_postgres_database.models.{p.name.removesuffix('.py')}"
+        )
+
+
+def create_erd(image_path: Path, include_table_names: list[str] | None = None):
+    """
+    create the pydot graph object by autoloading all tables via a bound metadata object
+    """
+
+    kwargs: dict[str, Any] = {
+        "show_datatypes": True,  # The image would get nasty big if we'd show the datatypes
```

Comment:
Should this be set to `True`? From the comment it looks like it should be set to `False`...

Reply:
I like "nasty big" images :-) This can be tuned afterwards ... for the moment I would like to have a working tool